ibis/expr/operations/relations.py

```diff
@@ -408,29 +408,40 @@ class SQLQueryResult(Relation):
 
 
 @public
-class SQLStringView(PhysicalTable):
-    """A view created from a SQL string."""
+class View(PhysicalTable):
```

Comment:
Could we rename this to something like `NamedRelation`? Possibly that should be a base class for `PhysicalTable(NamedRelation)`.

Reply:
I'd rather not do that in this PR, it's not directly related to the problem.